app/controllers/api/v1/deletions_controller.rb

```diff
@@ -13,7 +13,11 @@ def create
       render text: "Successfully deleted gem: #{@version.to_title}"
     else
       StatsD.increment 'yank.failure'
-      render text: "The version #{params[:version]} has already been deleted.",
+      error_messages = ""
+      @deletion.errors.full_messages.each do |msg|
+        error_messages = error_messages + msg + "\n"
+      end
+      render text: error_messages,
```

Comment:
I think this is probably better as `render text: @deletion.errors.full_messages.to_sentence`.